.evergreen/generate_evergreen_tasks.js

```diff
@@ -505,18 +506,6 @@ SINGLETON_TASKS.push(
         { func: 'run lint checks' }
       ]
     },
-    {
-      name: 'run-resource-management-no-async-dispose',
-      tags: ['resource-management'],
-      commands: [
-        updateExpansions({
-          NODE_LTS_VERSION: 'v16.20.2',
-          NPM_VERSION: 9
-        }),
-        { func: 'install dependencies' },
-        { func: 'check resource management' }
-      ]
-    },
```

Comment:
I can't comment on line 641 but should we remove `NPM_VERSION: 9` from the custom fle tests as well?